src/Guard/Guard.js

```diff
@@ -1,10 +1,12 @@
-import useFetch from '../useFetch';
 import isNormal from '../isNormal';
+import useData from '../useData';
+import useStatus from '../useStatus';
 
-function Guard({ fetch, children }) {
-  const [data, status] = useFetch(fetch);
+function Guard({ fetch, children, shouldShowWhen = isNormal }) {
+  const data = useData(fetch);
+  const status = useStatus(fetch);
 
-  return isNormal(status) && children(data);
+  return shouldShowWhen(status) && children(data);
```

Comment:
Didn't we agree to leave out the `shouldShowWhen` param?

Reply:
I think we did. I forgot tbh. I can remove it before it hits master

Reply:
done.